en/docs/design/advanced-topics/validate-api-definitions-with-linters.md

```diff
@@ -0,0 +1,93 @@
+# Validating API Definitions Against Custom Linter Rules
+
+By default, API Definitions are validated against default OpenAPI schema when importing and writing or editing them inside the publisher portal. Apart from the default OpenAPI validation,  the APIs created by the API publisher can be validated against a set of custom rules and it is useful in standardizing and keeping API definitions clean.
```

Comment:
Additional space after comma-> OpenAPI validation,  the

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6531/commits/d4f73e17e615acdef46fba31312245a0fe4a3a39